### PR TITLE
Added TylerGuthrie_weightConverter.py with purposes of documenting

### PR DESCRIPTION
**Pull request by tylereg03**
This is my second contribution to the ChatGPT Lab. This file is the code referenced in issue #79. This has not been well-documented yet. Again, I am seeking for ChatGPT to assist me in documenting this code, as the code in the current moment can be hard to follow as functions and error handlers are not properly described and segmented.

### CODE REVIEW

**Comment by tylereg03** [comment]
After my request to document the given code, ChatGPT made the code much more understandable by simply adding some comments throughout the code. For one, it split up all the conversion functions by unit automatically. It also explained what the error handling was meant for, and what it was doing, making it easier to understand. Finally, it specified when the code entered the printing phase. Considering that this code snippet was much harder to follow due to the line count and more complex error checking, ChatGPT's documentation suggestions were very helpful. The conversation that I had with ChatGPT can be found below. Closes #79.

Conversation: https://chat.openai.com/share/a1157789-5e35-4249-b103-b2cd9d9567cc

**Review by VyProductions — APPROVED** [review]
